src/stylesheets/components/_accordions.scss

```diff
@@ -81,6 +82,31 @@ $accordion-border: 3px solid $color-gray-lightest;
     > li {
       @include accordion-list-item-styles;
     }
+  }
+
+  [aria-expanded=false] {
```

Comment:
I'm wondering if this was just a rebase error that mashed the old accordion code with your refactor PR? I can't remember changing these.